modules/local/gens/main.nf

```diff
@@ -0,0 +1,51 @@
+process GENS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container 'docker.io/raysloks/gens_preproc:1.0.1'
+
+    input:
+    tuple val(meta), path(read_counts)
+    path  vcf
+    path  gnomad_positions
+
+    output:
+    tuple val(meta), path('*.cov.bed.gz'), emit: cov
+    tuple val(meta), path('*.baf.bed.gz'), emit: baf
+    path  "versions.yml"                 , emit: versions
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def avail_mem = 6
+    if (!task.memory) {
```

Comment:
Don't think it is needed here? Or is there something weird going on with the Perl script

Reply:
You're right. Its not necessary here. Will remove it 😄